### ISSUE TITLE
don't output csv

### ISSUE THREAD

**Issue by upszot** [davidB/kubectl-view-allocations#185]
oc view-allocations  -o csv

```
thread 'main' has overflowed its stack
fatal runtime error: stack overflow
Aborted (core dumped)
```

**Comment by davidB** [comment]
What is `oc` ?

**Comment by upszot** [comment]
OpenShift CLI (oc)

https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html

**Comment by davidB** [comment]
fix in 0.15
